### PR TITLE
Added ids for cypress

### PR DESCRIPTION
**Pull request by Camelia-Orcid**

### CODE REVIEW

**Comment by Camelia-Orcid** [comment]
[Automation Support - Ids required for cypress scripts](https://trello.com/c/OeGJ7fcP/8020-automation-support-ids-required-for-cypress-scripts)

File: src/app/cdk/panel/panels/panels.component.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export class PanelsComponent implements OnInit {
     label: string
     modal?: ComponentType<any>
     type?: EXTERNAL_ID_TYPE_WORK
+    id?: string
```
Comment (amontenegro) [inline src/app/cdk/panel/panels/panels.component.ts:60]: @Camelia-Orcid why do you added this id here? looks like you are not using it anywhere

**Review by amontenegro — CHANGES_REQUESTED** [review]

File: src/app/cdk/panel/panels/panels.component.ts (hunk 1 above)
Comment (Camelia-Orcid) [reply src/app/cdk/panel/panels/panels.component.ts:60]: @amontenegro  https://github.com/ORCID/orcid-angular/pull/1559/files#diff-e6bf5cd98d0dda8e73fa7e5b9924a06df2f53fc27031808b07f2d4971e94f3d6R135

*ngFor="let option of addMenuOptions"
            (click)="add('works', option.action)"
            id="{{ option.id }}"

action and id are attribuites or option object  stored in addMenuOption array

**Review by amontenegro — APPROVED** [review]

**Comment by leomendoza123** [comment]
[[QA] Search for trusted individual by email address not working as expected](https://trello.com/c/2MTFpYQz/8038-qa-search-for-trusted-individual-by-email-address-not-working-as-expected)

**Comment by leomendoza123** [comment]
[[QA] Search for trusted individual by email address not working as expected](https://trello.com/c/2MTFpYQz/8038-qa-search-for-trusted-individual-by-email-address-not-working-as-expected)

**Pull request merged**